ansible/roles/stack-sunbird/defaults/main.yml

```diff
@@ -875,23 +875,26 @@ uci_outbound_liveness_readiness:
 
 uci_transformer_liveness_readiness:
   healthcheck: true
-  readinessProbe:
+  livenessProbe:
+    failureThreshold: 2
     httpGet:
-      path: /service/health
+      path: /health
       port: 9091
-    initialDelaySeconds: 15
-    periodSeconds: 15
+      scheme: HTTP
+    initialDelaySeconds: 90
```

Comment:
Any reason why the initial delay needs to be so long? Usually we keep it short like 15s including the periodic probing

Reply:
Yes. The service downloads 200-300 small files from a different server and caches it on its end. That takes around 30-40 seconds. I have kept a buffer since the number of files will keep increasing.

Reply:
Different service = ODK service.